src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let oddReturn = 0;
+
+  function sum(num1) {
```

Comment:
You don't need to name functions if you return them right away.

```suggestion
  return (num1) => {
```